ansible/roles/multifaucet/templates/multifaucet/faucet.conf.php

```diff
@@ -3,10 +3,10 @@
 $config = array(
         "mysql_table_prefix" => getenv("MULTIFAUCET_DB_PRFX"), // table prefix to use
 
-        "minimum_payout" => 1000, // minimum to be awarded
-        "maximum_payout" => 1000, // maximum to be awarded
-        "payout_threshold" => 1000, // payout threshold, if the faucet contains less than this, display the 'dry_faucet' message
-        "payout_interval" => "1m", // payout interval, the wait time for a user between payouts. Type any numerical value with either a "m" (minutes), "h" (hours), or "d" (days), attached. Examples: 50m for a 50 minute delay, 7h for a 7 hour delay, etc.
+        "minimum_payout" => getenv("MULTIFAUCET_MIN_PAYOUT"), //1000, // minimum to be awarded
```

Comment:
Let's keep default value if env var is empty

Reply:
It is "defaulted" from ansible code, and we are not planning to run it outside of ansible.
Anyway is there a simple way to do that in php?

Reply:
I would do that on both sides. Just use ternary operator and if env variable is empty.

Since PHP 5.3 you can do:
```php
"minimum_payout" => getenv("MULTIFAUCET_MIN_PAYOUT") ?: 1000, 
```